ocpmodels/common/utils.py

```diff
@@ -551,12 +551,12 @@ def radius_graph_pbc(
     batch_size = len(data.natoms)
 
     if hasattr(data, "pbc"):
-        data.pbc = np.array(data.pbc)
+        data.pbc = torch.array(data.pbc)
```

Comment:
When you are creating a new torch array, you would want to create it in the device `torch.array(data.pbc, device=device)`

Reply:
Decided to just make data.pbc a tensor in the first place (in AtomsToGraphs) so  we shouldn't have to do that here